README.md

```diff
@@ -139,6 +154,26 @@ __AWESOME__ apps counter: __111_ 🎉
 - [x] [GitHub](https://github.com/tachiyomiorg/tachiyomi)
 - [x] [Official page](https://tachiyomi.org/)
 
+### TachiyomiSY
+
+<img alt="TachiyomiSY" height="64" src="https://github.com/jobobby04/TachiyomiSY/blob/master/app/src/main/ic_launcher-web.png?raw=true">
+
+- [ ] Google Play
+- [ ] F-Droid
+- [x] [GitHub](https://github.com/jobobby04/TachiyomiSY)
+- [ ] Official page
+
+## App Backup
+
+### Neo Backup
```

Comment:
When I installed this app on my smartphone it instantaneously crashes on the start. Could you relate to this? However apps looks awesome!

Reply:
Weird I used it recently to backup some apps a couple of days ago,

> Sorry for long wait. I saw your huge commit and to be honest still don't know how to review it. Could you please help by describing each of these apps that you have listed? Here is a template:
> 
> ```gfm
> ### AppName
> - [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
> - [x] It has 0 trackers on [εxodus](exodus report link).
> - [x] It has 0 F-Droid [Anti-Features](F-Droid / IzzyOnDroid).
> - [x] Has dark theme (if it doesn't, then app cannot be merged).
> - [x] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart. (optional, only when you put the heart next to the app)
> ```

Where should I put this?

Reply:
Tested this on Android 11 and 12 (both custom ROMs) I think it crashes since you have to give it root permissions.

Reply:
Oh... that is why... well I appreciate the effort, but I think most "casual" users won't know this and won't have any root permissions. But still I do not want to put it in some `## Root` category. I am thinking about a quick note that it requires a root to use it.